homeassistant/components/sensor/darksky.py

```diff
@@ -97,6 +98,14 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         _LOGGER.error("Latitude or longitude not set in Home Assistant config")
         return False
 
+    # Check to make sure that forecast is in a valid range of 1 week
+    forecast = config.get(CONF_FORECAST)
```

Comment:
I'm not quite sure how the vol.Range works, it is throwing an error for me. It appears it is trying to compare a single value against the range, but I have a list of days, not a single day. I'll have to investigate more when I have time.